extensions/blocks/social-previews/modal.scss

```diff
@@ -0,0 +1,158 @@
+/**
+ * Modal styles for Social Previews
+ */
+
+@import '../../shared/styles/gutenberg-base-styles.scss';
+
+// Styles shared between modals
+.jetpack-social-previews__modal {
+	.components-modal__header {
+		margin: 0;
+	}
+
+	.components-modal__content {
+		padding: 0;
+	}
+}
+
+// Main modal containing the social previews
+.jetpack-social-previews__modal-previews {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	.components-tab-panel__tabs {
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		padding: 12px;
+		max-width: none;
+
+		.components-button {
+			outline: 0;
+			margin: 3px 0;
+			font-size: 0;
+
+			svg {
+				display: block;
+				fill: currentColor;
+			}
+
+			&.is-active {
+				box-shadow: 0px 0px 0px 2px $blue-medium-focus;
+			}
+
+			&:not( :disabled ):not( [aria-disabled='true'] ):not( .is-secondary ):not( .is-primary ):not( .is-tertiary ):not( .is-link ):hover {
+				box-shadow: 0px 0px 0px 2px $blue-medium-focus;
+			}
+		}
+
+	}
+
+	.components-tab-panel__tab-content {
+		padding: 10px;
+		background-color: $light-gray-100;
+		flex: 1;
+	}
+
+	@media ( min-width: $break-medium ) {
+		flex-direction: row;
+		width: 900px;
+		min-height: 500px;
+
+		.components-tab-panel__tabs { 
+			flex-direction: column;
+			justify-content: flex-start;
+			padding: 24px;
+
+			.components-button {
+				font-size: 13px;
+
+				> svg {
+					margin-right: 8px;
+				}
+			}
+		}
+
+		.components-tab-panel__tab-content {
+			padding: 40px;
+		}
+	}
+}
+
+// Modal containing the upgrade nudge
+.jetpack-social-previews__modal-upgrade {
+	padding: 2em;
+
+	.jetpack-social-previews__upgrade-illustration {
+		width: 100%;
+		max-width: 351px;
+		height: auto;
+	}
+
+	.jetpack-social-previews__upgrade-description {
+		margin-bottom: 1em;
+	}
+	
+	.jetpack-social-previews__upgrade-heading {
+		font-size: 2em;
+		line-height: 1.15;
+	}
+	
+	.jetpack-social-previews__upgrade-feature-list {
+		list-style: none;
+		margin-bottom: 2em;
+		padding-left: 1em;
+		font-size: 1.1em;
+		line-height: 1.4;
+	
+		li {
+			position: relative;
+			margin-bottom: 12px;
+	
+			&:before {
+				content:  "\2713 ";
+				position: absolute;
+				left: -20px;
+				color: $alert-green;
+			}
+		}
+	}
+
+	@media ( min-width: $break-small ) {
+		width: 80vw;
+		max-width: 870px;
+		display: grid;
+		grid-gap: 3em;
+		grid-template-columns: 1fr 1fr;
```

Comment:
I didn't realize we could use grid! 👍 I didn't know if support was there enough for it.